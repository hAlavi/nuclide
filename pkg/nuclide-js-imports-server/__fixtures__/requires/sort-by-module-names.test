const p = require('C');
const q = require('A');
const r = require('b');
const U = require('b');
const V = require('E');
const W = require('D');
import type X from 'b';
import type Y from 'C';
import type Z from 'A';
require('A');
require('b');
require('C');

const m: [X, Y, Z] = p(q(r(U, V, W)));
