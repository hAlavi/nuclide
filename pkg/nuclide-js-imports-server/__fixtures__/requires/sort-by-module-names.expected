import type Z from 'A';
import type Y from 'C';
import type X from 'b';

require('A');
require('C');
require('b');

const q = require('A');
const p = require('C');
const W = require('D');
const V = require('E');

const r = require('b');

const m: [X, Y, Z] = p(q(r(U, V, W)));
